src/helpers/loginHelpers.js

```diff
@@ -0,0 +1,8 @@
+import * as requestHelpers from './requestHelpers';
+import * as configuration from './../constants/configuration.constants';
+
+export const login = async (email, password) => {
+    const result = await requestHelpers.postData(`${configuration.DEV_SERVER_URL}/api/login`, {email: email, password: password});
```

Comment:
I think we should use envinroment variables to check which URL we should use, because this code deployed will not work right now.